File: src/app/src/main/java/com/feng/digitacoin/ui/me/ChooseDateActivity.java
package com.feng.digitacoin.ui.me;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;
import android.support.v7.widget.RecyclerView;

import com.feng.digitacoin.R;

/**
 * @author weixuefeng@lubangame.com
 * @version $Rev$
 * @time: 2018/11/20--2:56 PM
 * @description
 * @copyright (c) 2018 Newton Foundation. All rights reserved.
 */
public class ChooseDateActivity extends AppCompatActivity {

    private RecyclerView recyclerView;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_choose_date);
        recyclerView = findViewById(R.id.dateRecycler);
    }
}

File: src/app/src/main/java/com/feng/digitacoin/ui/example/ChooseDateActivity.kt
package com.feng.digitacoin.ui.example

import android.os.Bundle
import com.feng.digitacoin.R
import com.feng.digitacoin.extension.loge
import com.feng.digitacoin.ui.BaseActivity

/**
 * @author weixuefeng@lubangame.com
 * @version $Rev$
 * @time: 2018/11/20--2:56 PM
 * @description
 * @copyright (c) 2018 Newton Foundation. All rights reserved.
 */
class ChooseDateActivity : BaseActivity<DateContract.View, ChoosePresenter>(), DateContract.View {

    override var mPresenter: ChoosePresenter = ChoosePresenter()

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_choose_date)
        mPresenter.loadListData()
        showError("error")
    }

    override fun showList(list: List<Int>) {
        loge(list.toString())
    }
}
